components/table/table-wrapper.js

```diff
@@ -62,6 +63,15 @@ export const tableStyles = css`
 	th.d2l-table-header-col-sortable {
 		padding: 0;
 	}
+	.d2l-checkbox,
+	d2l-input-checkbox,
+	d2l-selection-select-all,
+	input[type="checkbox"] {
+		margin-block: calc(-0.5 * (${cssSizes.inputBoxSize}rem - var(--d2l-table-cell-padding)));
```

Comment:
Not sure I'm understanding the reasoning behind this calculation, and the results don't quite seem correct (e.g., on the test page the button icon spacing isn't quite right).

Think more about what you're looking to correct for. 
Without any change (i.e., prior to this PR), the cell height = checkbox height + padding (with medium font-size this is 24px + 15px + 15px = 54px)
We want the cell height to be 48px. So how much less does the padding need to be in order for this to be the case? Is there a way we can consistently calculate that using variables?